File: widgets/bridge_plugin/android/src/main/kotlin/com/facephi/bridge_plugin/sdkmanager/data/response/SdkResponse.kt
package com.facephi.bridge_plugin.sdkmanager.data.response

data class SdkResponse(
    var  finishStatus : Int = FinishStatus.STATUS_ERROR,
    var  errorType : Int = ErrorType.TE_UNKNOWN_ERROR,
    var selphiResponse: SelphiResponse? = null,
    var selphIdResponse: SelphIdResponse? = null
)

File: Sources/facephiSDK-ios-package/Response/SdkResponse.swift
//
//  SdkResponse.swift
//
//
//  Created by Faustino Flores García on 15/12/21.
//

import Foundation

public struct SdkResponse: Codable {
    public var finishStatus = FinishStatus.STATUS_ERROR.rawValue
    public var errorType = ErrorType.TE_UNKNOWN_ERROR.rawValue
    public var selphiResponse: SelphiResponse? = nil
    public var selphIdResponse: SelphIdResponse? = nil

    public init(withErrorType: Int) {
        self.errorType = withErrorType
    }

    public init(withFinishStatus: Int) {
        self.finishStatus = withFinishStatus
    }

    public init(withFinishStatus: Int, andErrorType: Int) {
        self.finishStatus = withFinishStatus
        self.errorType = andErrorType
    }

    public init() {}
}
